auth-web/src/components/auth/staff/continuation-application/AuthorizationContactInformation.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    v-if="!!review && !!filing"
+    id="authorization-contact-information"
+  >
+    <!-- Authorization Contact Information -->
+    <section class="section-container">
+      <v-row no-gutters>
+        <!-- First Column: Contact Information Label -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Contact Information</label>
+        </v-col>
+        <!-- Second Column: Email Address Label and Value -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Email Address</label>
+          <div>{{ email || '[Unknown]' }}</div>
+        </v-col>
+        <!-- Third Column: Phone Number Label and Value -->
+        <v-col
+          cols="12"
+          sm="4"
+        >
+          <label>Phone Number</label>
+          <div>{{ phone || '[Unknown]' }}</div>
+        </v-col>
+      </v-row>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { ContinuationFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
+import { computed, defineComponent, reactive } from '@vue/composition-api'
+
+export default defineComponent({
+  name: 'AuthorizationContactInformation',
+
+  props: {
+    review: { type: Object as () => ContinuationReviewIF, required: true },
```

Comment:
What do you use this prop for?

Reply:
I saw other sections used this prop, only on line 3 to check if the review is not null

Reply:
The conditional code you saw is to not render the page unless/until the data is present (since a fetch may be in progress). This is specifically to prevent the need to check if the object exists before accessing its properties.

In this case, since this component doesn't use this object, I think you can remove the prop. A better answer: test without it and make sure everything displays nicely :)

Reply:
Tested and removed the prop for review, not needed